change/@ni-eslint-config-typescript-67da7439-6c61-46c1-9440-60866dfdaae9.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "none",
```

I think removing comments alone is not considered a `patch` version bump. Please update the `dependentChangeType` to `none` as well if you agree.

---

I'm actually not certain which configuration wins if a library specifies `"type": "none"` but a dependency specifies `"dependentChangeType": "major"` (as we do in the eslint-config-javascript changefile). But I would vote to make this a major change to all libraries so think it would be most clear to make this changefile more closely match the javascript one: use `major` change type and set the comment to `"Remove Airbnb dependency from style guide"`

---

I will update this based on the decision taken from the other thread.

---

Updated the type and dependentChangeType as "patch.